src/webui/api/authcontroller.h

```diff
@@ -54,7 +55,7 @@ private slots:
     struct FailedLogin
     {
         int failedAttemptsCount = 0;
-        qint64 bannedAt = 0;
+        QDeadlineTimer bannedAt {-1};
```

Comment:
Last comment.
Seems this field has inappropriate name now. Previously it meant the time point the user is banned at. Currently it isn't.

Reply:
Renamed to `banTimer`.